src/UTF-unknown.csproj

```diff
@@ -1,17 +1,19 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.0;netstandard1.3;net40</TargetFrameworks>
+    <TargetFrameworks>netstandard1.0;netstandard1.3;net40;netcoreapp3.0</TargetFrameworks>

```

Comment:
Please also update the unit test project

Reply:
I also tried to add, but for some reason the tests were not performed. Maybe something did wrong. But, in Core 3.0, there will be throw exception by this
https://github.com/CharsetDetector/UTF-unknown/blob/a27750757884a3104bf840995a04277a2709a1a1/tests/CharsetDetectorTestBatch.cs#L112

Because not everything is implemented

Reply:
Fixed